frontend/packages/integration-tests-cypress/tests/app/overview.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { Set as ImmutableSet } from 'immutable';
+import { safeLoad, safeDump } from 'js-yaml';
+import * as _ from 'lodash';
+
+import { nav } from '../../views/nav';
+import * as yamlEditor from '../../views/yaml-editor';
+import { overviewPage } from '../../views/overview';
+import { detailsPage } from '../../views/details-page';
+import { errorMessage } from '../../views/form';
+import { masthead } from '../../views/masthead';
+
+import { checkErrors, testName } from '../../support';
+import {
+  DeploymentModel,
+  StatefulSetModel,
+  DeploymentConfigModel,
+  DaemonSetModel,
+} from '../../../../public/models';
+
+const overviewResources = ImmutableSet([
+  DaemonSetModel,
+  DeploymentModel,
+  DeploymentConfigModel,
+  StatefulSetModel,
+]);
+
+describe('Visiting Overview page', () => {
+  afterEach(() => {
+    checkErrors();
+  });
+
+  before(() => {
```

Comment:
Small nit.  typically it goes
- before (all)
- beforeEach
- afterEach
- after (all)

Just for readability.  -thanks

Reply:
@dtaylor113 done